samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/main.c

```diff
@@ -197,9 +197,15 @@ void main(void)
 	ps_settings_init();
 
 	/* Initialize the Bluetooth Subsystem */
-	err = bt_enable(bt_ready);
+	err = bt_enable(NULL);
 	if (err) {
 		printk("Bluetooth init failed (err %d)\n", err);
+		return;
+	} else {
+		bt_ready(err);
+		if (err) {
```

Comment:
err is 0 in this branch, so this test doesn't make sense

Reply:
you can also remove the err parameter from bt_ready since you're never passing anything else than 0

Reply:
Done !!